### PR TITLE
mrc-4678: Remove rehydrate from JSON feature

### PR DESCRIPTION
**Pull request by r-ash**
## Description

This feature has been hidden behind a feature switch and not used for past year or so now can rehydrate from the output zip. Removing this as no longer required!

## Type of version change
_Delete as appropriate_

None

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/src/app/hintVersion.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const currentHintVersion = "3.5.4";
+export const currentHintVersion = "3.6.0";
```
Comment (EmmaLRussell) [inline src/app/static/src/app/hintVersion.ts:1]: Arguably this isn't changing functionality so shouldn't need a new version... and if it is, shouldn't the news file be getting updated too?

**Review by EmmaLRussell — APPROVED** [review]
Nice!

File: src/app/static/src/app/hintVersion.ts (hunk 1 above)
Comment (r-ash) [reply src/app/static/src/app/hintVersion.ts]: Good point, I've reverted this version change. Forgot completely about the NEWS, will note that for other PRs

**Review by M-Kusumgar — APPROVED** [review]
LGTM! feels good to remove some lines from hint

**Pull request merged**